.jvmopts

```diff
@@ -8,5 +8,5 @@
 -XX:+TieredCompilation
 -XX:-UseGCOverheadLimit
 # effectively adds GC to Perm space
--XX:+CMSClassUnloadingEnabled
+#-XX:+CMSClassUnloadingEnabled
```

Comment:
I had to do this in order to get it to build on my system. According to the interwebs this flag might have been deprecated in new JVMs? This is obviously not required as part of the fix.